apps/web/lib/pollar/README.md

```
@@ -0,0 +1,87 @@
+
+## Feature flag
+
+```bash
+NEXT_PUBLIC_ENABLE_POLLAR_ONBOARDING=true
+NEXT_PUBLIC_POLLAR_PUBLISHABLE_KEY=pub_testnet_...
+POLLAR_SECRET_KEY=sec_testnet_...   # server-only
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)